docs/ops/logging/troubleshooting.rst

```diff
@@ -0,0 +1,31 @@
+#############################
+Troubleshooting Elasticsearch
+#############################
+
+.. rubric:: Memory limits
```

Comment:
For this page you can switch over to regular sectioning, e.g.

```rst
Memory limits
=============
```

The only reason I'm using `rubic` on the individual app homepages is to prevent subsections from going into the toctree, though in retrospect I think this could be accomplished other ways...